V5_MIGRATION.md

```diff
@@ -16,23 +16,58 @@ v5 introduces limited support for Swift Package Manager. See the [README](/READM
 
 In v4, 3D Secure classes were housed in the `BraintreePaymentFlow` module. In v5, `BraintreeThreeDSecure` is a standalone module offering the same 3DS functionality. The `BraintreePaymentFlow` module still houses Local Payment functionality.
 
-### CocoaPods
+#### Integration
+
+<details><summary>CocoaPods</summary>
+<p>
```

Comment:
Nitpick - I don't think we need collapsable sections. I haven't seen them in markdown before and _maybe_ a merchant will miss them. Also there isn't too many lines in each section.

Reply:
For me, having the collapsible sections really helps the readability of the entire doc. If you don't have a strong preference, I think I'll leave them in.